cleaning_services/demo/account_analytic_account.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="account_analytic_account_2" model="account.analytic.account">
```

Comment:
Same as for worksheet, analytic account are automatically handled by project and invoice.
What could be useful, though, is to activate the group (`group_analytic_accounting`) to handle the full potential of analytic.